src/System.Private.ServiceModel/src/System/ServiceModel/Channels/HttpChannelFactory.cs

```diff
@@ -100,7 +100,9 @@ internal HttpChannelFactory(HttpTransportBindingElement bindingElement, BindingC
             TransferMode = bindingElement.TransferMode;
             _keepAliveEnabled = bindingElement.KeepAliveEnabled;
 
-            if (bindingElement.ProxyAddress != null)
+            if (bindingElement.Proxy != null)
+                _proxy = bindingElement.Proxy;
```

Comment:
Coding style requires if statements to be in a code block
```suggestion
            {
                _proxy = bindingElement.Proxy;
            }
```